feat: changed file_exists to suitable is_dir/is_file in core namespace core (#10832)
shopware/platform: changelog/_unreleased/2025-06-26-changed-file_exists-to-suitable-is_dir-is_file.md

---

> title: Changed file_exists to suitable is_dir/is_file

Reviewer: ```suggestion
title: Remove usage of file_exists
```

issue: 5913
author: tinect
author_email: s.koenig@tinect.de
author_github: tinect
---

# Core

> * Changed usage file_exists to suitable is_dir or is_file

Reviewer: ```suggestion
* Changed usage of `file_exists` to better suiting functions `is_dir` or `is_file`
```
